website/public/js/services/taskServices.js

```diff
@@ -6,11 +6,11 @@ angular
 
 tasksFactory.$inject = [
   '$rootScope',
-  'User',
-  'Shared'
+  'Shared',
+  'User'
 ];
 
-function tasksFactory($rootScope, User, Shared) {
+function tasksFactory($rootScope, Shared, User) {
```

Comment:
Going forward, we're trying to follow a specific order for these injections. Angular services go first, then custom services, alphabetically.